app/src/main/java/com/nextcloud/talk/webrtc/PeerConnectionWrapper.java

```diff
@@ -256,6 +258,15 @@ public boolean isMCUPublisher() {
         return isMCUPublisher;
     }
 
+    private boolean isAudioCallOnly() {
```

Comment:
Rather than `isAudioCallOnly()` it might be better to name it something like `shouldReceiveVideo()`. If the PeerConnectionWrapper is the sender peer to the HPB [`OfferToReceiveVideo` would be false too](https://github.com/nextcloud/talk-android/blob/1b30a6ef62dde17b9eb95a6867d03089a5b3dea9/app/src/main/java/com/nextcloud/talk/activities/CallActivity.java#L447), so in that case `isAudioCallOnly()` would return `true` even in a video call.

Right now this should not be a problem, as `isAudioCallOnly()` is used only from `onSetSuccess()` if there is no local description, so in practise it will not be called for the sender peer to the HPB. But if the method is named `isAudioCallOnly()` it could be misused in the future :-)